### DISCUSSION TITLE
Could you tell me when I can register

### DISCUSSION THREAD

**Discussion by ghost** [community/community#15866] [Copilot Conversations] [answered]
Could you tell me when I can register

**Comment by davecheney** [comment, marked as answer]
Registrations are open, https://copilot.github.com/

**Reply by ghost** [reply]
I didn't get permission, and Tell me I need to wait

**Reply by ghost** [reply]
TIPS:You’re already on the waitlist for GitHub Copilot! We’ll notify you when we’ve enabled it on your account. Make sure your [primary email address is up-to-date](https://github.com/settings/emails) so we can notify you.
